data/human/human missions.txt

```diff
@@ -4339,3 +4339,289 @@ mission "Saving Artifacts 4"
+			`	The result is not what you were hoping for. The display maps two leviathan-sized things ahead, each over 200 meters long. Dozens of sets of flippers run along each 80-meter-tall side and hundreds of teeth the size of an adult human line their jaws.`
+			`	You goggle at the display for a moment, taking it in, and draw breath to call out the alarm when an eye appears through the ship window, yellow, glowing, alien, and massive. All at once, you turn and accelerate, shouting the alarm to any who hadn't already realized, and hoping that you're leaving the leviathan beasts behind. A quick glance at the display shows at least one subpod following behind. Good enough. If one of those rams the ship, nobody is going home anyway.`
+			`	It's a long while before you slow and turn around. No mammoth sea monster comes out of the darkness, and you take a breath.`
+			`	"Quoijuu," says Karengo simply.`
+				goto failure
+			label east4
+			`	The proximity alarm wails and this time, you're much quicker to stop the ship... but your tracker shows two subpods smash straight into the stone wall looming in front of you.`
+			`	Again, you put the ship into reverse and turn around. This is not going at all how it was supposed to.`
+			label failure
+			`	Jorma and Daciana never report back in.`
+			`	"You stay," radios Karengo. He sounds frustrated for the first time, and you keep the ship in place, watching his subpod dot radiate outwards, exploring.`
+			`	"Come here." he finally says. His dot has stopped moving, hovering to the south, and you turn to follow, wondering if this misadventure is finally over.`
+			`	"It's here," radios Karengo. "Fire off a ping." You comply, and the display shows a massive object directly below you.`
+			label treasure
+			`	A 3D holo floats on the console, displaying the smooth lines of a starship soaring down into the depths. The jagged edges suggest that this is only a part of a much larger vessel, although it's hard to guess at the size of the whole ship from the fragment.`
+			`	Your craft moves closer to investigate and you send out another ping, this time more focused, and the holographic ship display flickers and refines. The outlines of sealed entrances are visible, unnecessary with the ship torn apart as it is. Sea growth is mostly absent from the hull, to your surprise, except for a radial pattern of some sort of crystal-shaped growth. Infrared imaging shows a dim glow of heat at its center.`
+			branch somesurvive
+				or 
+					has "Mussie dead" 
+					has "Elias and Taddese dead"
+					has "Jorma and Daciana dead"
+			branch alldead
```

Comment:
I selected east, west, and north, which results in all of the passengers dying, yet triggered `somesurvive` instead of `alldead`. Maybe conditions missing somewhere?

Reply:
To be honest, I don't really know what's causing that. I checked all the choices that result in someone dying, and I found one that didn't have the action that set the `has "someone dead"` label. I fixed that, but it wasn't one of the selections you mentioned.

Reply:
Hmm. The only other thing I could think of is it's checking `somesurvive` before `alldead` and thus is doing `somesurvive` all the time because you'll meet those conditions if you have `alldead` - so maybe the branch checks should swap order? But I tried that and didn't get any different results. I don't have time to do more testing at the moment, but let me know if you figure it out.

Reply:
I tried fixing it with the help of becca and todd, it should work now, but I'm not sure.

Reply:
It works! I finally managed to playtest it and the branching works!

Reply:
No, now it's just always going to `alldead`.

Reply:
Ok, looks like it was an issue with my save file. I've playtested it properly and it should work now.